loxone.go

```diff
@@ -101,6 +101,16 @@ type Control struct {
 	Details    ControlDetails
 }
 
+// GetControl returns the control with the given uuid
+func (cfg *Config) GetControl(uuid string) *Control {
+	for key, control := range cfg.Controls {
+		if key == uuid {
+			return control
+		}
+	}
+	return nil
+}
```

Comment:
`make format` :)

Reply:
Hm, did that.. 🤷‍♂️
Do you want a line-break here?